### PR TITLE
Add prefer_related_applications

### LINKED ISSUE #1548: Fix manifest.pl
We need to add a new parameter to manifest.pl

https://developers.google.com/web/fundamentals/app-install-banners/native

`prefer_related_applications >needs to be> true`

### PR DESCRIPTION
**Pull request by PrajwalM2212**
fixes #1548 

**Checklist:**

Make sure you've done all the following (You can delete the checklist before submitting)

* [ ] Code is well documented
* [ ] Include unit tests for new functionality
* [ ] Code passes Travis builds in your branch
* [ ] If you have multiple commits please combine them into one commit by squashing them.
* [ ] Read and understood the [contribution guidelines](https://github.com/openfoodfacts/openfoodfacts-server/blob/master/CONTRIBUTING.md)

**Description:**

Describe the changes made and why they were made instead of how they were made.

**Related issues and discussion:** #[ISSUE NUMBER]

### CODE REVIEW

**Comment by PrajwalM2212** [comment]
This is my first perl PR .

File: cgi/manifest.pl [hunk 1]
```diff
@@ -65,6 +65,7 @@
 $manifest{start_url} = format_subdomain($subdomain);
 $manifest{scope} = '/';
 $manifest{display} = 'standalone';
+$manifest{prefer_related_applications} = true;
```
Comment (stephanegigandet) [inline cgi/manifest.pl]: This needs to be 'true' instead of true to make it a string, there is no boolean type in Perl.

**Comment by teolemon** [comment]
@stephanegigandet good to merge ?

File: cgi/manifest.pl (hunk 1 above)
Comment (hangy) [reply cgi/manifest.pl]: I think it should probably be `$JSON::PP::true`.

File: cgi/manifest.pl [hunk 2]
```diff
@@ -65,6 +65,7 @@
 $manifest{start_url} = format_subdomain($subdomain);
 $manifest{scope} = '/';
 $manifest{display} = 'standalone';
+$manifest{prefer_related_applications} = 'true';
```
Comment (hangy) [inline cgi/manifest.pl]: I think it should probably be `$JSON::PP::true`.

**Pull request merged**